README.md

```diff
@@ -1,34 +1,115 @@
+
 - GPS is used to find nearby restaurants, museums, and activities based on the user's current location.
 - The camera is used to change the trip's thumbnail, change profile picture and add shared pictures.
-- 
-### Offline mode
+-
+
+#### Offline mode
+
 - Previously planned schedules are still available offline, allowing for uninterrupted access even in areas with poor connectivity.
 - The search for users and the feed still work even without connexion, though a bit slower
 - When the user goes offline a red banner will be displayed until they reconnect and certain functions are disabled
-- Photos for users and trips are cached so available offline.
\ No newline at end of file
+- Photos for users and trips are cached so available offline.
+
+## Technical Summary:
+
+#### Tech Stack:
+
+- Frontent: JetpackCompose
+- Backend: Firestore
+- Auth: Firebase Auth
+- Sotrage: Firebase Cloud Storage
+- Other APIs: Google Maps and Places API (for location search and displaying maps), Gemini (for JSON schema generation)
+- CI/CD: Github Actions, SonarCloud (for code quality)
+
+#### Architecture:
+
+Architrecure type: MVVM; Diagram visible [here](https://github.com/swent-t16/Voyageur/blob/main/resources/architecture_diagram.png)
+
+##### View Models:
+
+- UserViewModel:
+
+  - ViewModel responsible for managing user data and interactions with Firebase authentication and user repository.
+  - It provides functionality to load, update, and observe user data, and to handle user-related actions such as signing in / out, adding/removing contacts, and searching users and listening for real time friend request updates.
+
+- TripViewModel:
+
+  - ViewModel for managing trip-related data and operations in the app.
+
+  - This ViewModel interacts with the `TripRepository` to fetch, create, update, and delete trips. It
+  - Handles the management of trip-related states such as the selected trip, selected day, activities, and UI state.
+  - It's responsible for handling file uploads to Firebase Storage and integrating with the AI assistant to generate trip activities.
+
+- PlacesViewModel:
+  - ViewModel for managing places-related data and operations in the app.
+  - This ViewModel interacts with the `PlacesRepository` to fetch places, search for places, and get place details.
+  - It handles the management of place-related states such as the selected place, search results, and UI state.
+
+##### Repositories:
+
+- Interfaces:
+
+  - `UserRepository`: Interface for managing user (basic CRUD operations) interactions with a data source.
+  - `TripRepository`: Interface for managing trip (basic CRUD operations) interactions with a data source.
+  - `PlacesRepository`: Interface for managing places-related interactions with a location (map-based) data source.
+  - `FriendRequestRepository` and `TripInviteRepository`: used managing friend requests and trip invites.
+
+- Repositproies (implementing the interfaces for Firestore and Google Maps API use cases):
```

Comment:
Here the word _Repositories_ is misspelled. Make sure to change it!

Reply:
Good catch, I will change it asap